CruxPay/Classes/CruxJSBridge.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  CruxJSBridge.swift
+//  CruxPay
+//
+//  Created by Sanchay on 14/11/19.
+//
+
+import Foundation
+import JavaScriptCore
+import CryptoSwift
+
+class CruxJSBridge {
+    lazy var context: JSContext? = {
+        let context = JSContext()
+        
+        let frameworkBundle = Bundle(for: type(of: self))
+        let bundleURL = frameworkBundle.resourceURL!.appendingPathComponent("CruxPay.bundle")
+        let resourceBundle = Bundle(url: bundleURL)
+        
+        guard let cruxJSPath = resourceBundle?.path(forResource: "cruxpay-0.1.5", ofType: "js"),
+            let requestDepsPath = resourceBundle?.path(forResource: "requestDeps", ofType: "js"),
+            let promiseDepsPath = resourceBundle?.path(forResource: "promiseDeps", ofType: "js") else {
+                print("unable to read resource files.")
+                return nil
```

Comment:
Raise error here if we can't read resource files.

Reply:
Call can throw, but errors cannot be thrown out of a property initializer

Initializing context here